exercises/state-of-tic-tac-toe/canonical-data.json

```diff
@@ -245,7 +245,7 @@
       "cases": [
         {
           "uuid": "b42ed767-194c-4364-b36e-efbfb3de8788",
-          "description": "Draw",
+          "description": "Draw (1)",
```

Comment:
This doesn't add any info nor does it explain why the case is there, I'd remove the numbering.

Uniqueness is not guaranteed or required in any way.

Reply:
I'm fine with changing the description to "another draw", and I would also push towards having unique descriptions, it really helps with generators, or in case of reimplementation to see which test needs to be changed, etc.

Reply:
That's redundant. It's obvious that in a test group that contains draws, as stated by the group description, all tests will be a draw or another draw. Unless there's an explicit reason for why a certain case exists, e.g. an edge case of a draw, this is useless info.

Reply:
(You could make an argument that there's no reason to have two drawn games that don't cover any particular edge cases in the first place but that ship has sailed now.)

Reply:
Well yes, a redundant naming convention for a redundant situation. The test case is fine BTW, maybe it will be crucial to a student who had an off-by-one error or something.

I don't really mind either way, I just want something unique so it's easier for people down the line.